drivers/docker_driver.go

```diff
@@ -53,7 +54,9 @@ func NewDockerDriver(args DriverConfig) (Driver, error) {
 }
 
 func (d *DockerDriver) Destroy() {
-	// noop
+	for _, image := range d.intermediateImages {
+		d.cli.RemoveImage(image)
```

Comment:
Maybe catch errors and log them?

Reply:
accidentally screwed up the history here but fixed